Core/include/Acts/Seeding/SpacePointGrid.ipp

```diff
@@ -14,27 +14,33 @@ template <typename SpacePoint>
 std::unique_ptr<Acts::SpacePointGrid<SpacePoint>>
 Acts::SpacePointGridCreator::createGrid(
     const Acts::SpacePointGridConfig& config) {
-  // calculate circle intersections of helix and max detector radius
-  float minHelixRadius = config.minPt / (300. * config.bFieldInZ);  // in mm
-  float maxR2 = config.rMax * config.rMax;
-  float xOuter = maxR2 / (2 * minHelixRadius);
-  float yOuter = std::sqrt(maxR2 - xOuter * xOuter);
-  float outerAngle = std::atan(xOuter / yOuter);
-  // intersection of helix and max detector radius minus maximum R distance from
-  // middle SP to top SP
-  float innerAngle = 0;
-  if (config.rMax > config.deltaRMax) {
-    float innerCircleR2 =
-        (config.rMax - config.deltaRMax) * (config.rMax - config.deltaRMax);
-    float xInner = innerCircleR2 / (2 * minHelixRadius);
-    float yInner = std::sqrt(innerCircleR2 - xInner * xInner);
-    innerAngle = std::atan(xInner / yInner);
-  }
+  int phiBins;
+  // for no (or bogus) magnetic field, create 100 phi-bins
+  if (config.bFieldInZ <= 0) {
```

Comment:
Negative values don't make sense any way? That's why you do `<= 0`, correct?

Reply:
Yep exactly. Do you think this should be handled separately/differently?

Reply:
Actually, a negative bfield would just be in the other direction so in principle I could leave it in. not sure if it would have implications in the code somewhere else. here I'd just have to take std::abs to make it work.

Reply:
I don't know. Can you test how the code behaves if `bFieldInZ < 0`? Otherwise, I'd maybe rather throw an error here?

Reply:
Negative values are now supported